var/spack/repos/builtin/packages/ncbi-rmblastn/package.py

```diff
@@ -26,11 +26,102 @@
 
 
 class NcbiRmblastn(AutotoolsPackage):
-    """RMBlast search engine for NCBI"""
+    """RMBlast search engine for NCBI."""
 
     homepage = "https://www.ncbi.nlm.nih.gov/"
     url      = "ftp://ftp.ncbi.nlm.nih.gov/blast/executables/rmblast/LATEST/ncbi-rmblastn-2.2.28-src.tar.gz"
 
     version('2.2.28', 'fb5f4e2e02ffcb1b17af2e9f206c5c22')
+    version('2.6.0', 'c8ce8055b10c4d774d995f88c7cc6225',
+            url='ftp://ftp.ncbi.nlm.nih.gov/blast/executables/blast+/2.6.0/ncbi-blast-2.6.0+-src.tar.gz')
+
+    patch('isb-2.6.0+-changes-vers2.patch', when='@2.6.0', level=1)
 
     configure_directory = 'c++'
+    build_directory = 'c++'
+
+    def configure_args(self):
+        if self.spec.version < Version('2.6.0'):
+            options = []
+        elif self.spec.version == Version('2.6.0'):
+            options = [
+                '--without-internal',
```

Comment:
it would probably be better to offer this as a variant. Even though I don't completely get what you are trying to achieve here, unconditionally deactivating all these is probably not a good idea and shouldn't most of these be dependencies?

Reply:
ncbi seems to be using the same build process as their main massive web enabled toolkit for the stand alone pieces. As such this links against a bunch of libraries that would really only be used in the web services ncbi toolkit. The blast programs have no need to be linked against kerberos or sasl for example.